### PR TITLE
EJ2-68151-off-Sample

### PR DESCRIPTION
**Pull request by SaravanaPriya31**

### CODE REVIEW

File: How to/Add water mark annotation in PDF viewer/Water mark annotation/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# MyApp
+# Sample to load office documents to PDF viewer
```
Comment (KameshRajendran) [inline How to/Add water mark annotation in PDF viewer/Water mark annotation/README.md:1]: why you have changed here?

File: How to/Add water mark annotation in PDF viewer/Web Service/Controllers/PdfViewerController.cs [hunk 2]
```diff
@@ -91,12 +91,54 @@ public IActionResult Load([FromBody] Dictionary<string, string> jsonObject)
             return Content(JsonConvert.SerializeObject(jsonResult));
         }
 
+        [AcceptVerbs("Get")]
```
Comment (KameshRajendran) [inline How to/Add water mark annotation in PDF viewer/Web Service/Controllers/PdfViewerController.cs:94]: why you have changed here?

File: office-products/Export success/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```
Comment (KameshRajendran) [inline office-products/Export success/.editorconfig:7]: what is the use of this file?

File: office-products/Export success/.browserslistrc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
```
Comment (KameshRajendran) [inline office-products/Export success/.browserslistrc:5]: what is the use of this file?

File: office-products/Export success/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# MyApp
```
Comment (KameshRajendran) [inline office-products/Export success/README.md:1]: is that correct?

File: office-products/Export success/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# MyApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.1.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
```
Comment (KameshRajendran) [inline office-products/Export success/README.md:7]: will it run when give run serve alone?

File: office-products/Export success/src/polyfills.ts [hunk 7]
```diff
@@ -0,0 +1,53 @@
+/**
+ * This file includes polyfills needed by Angular and is loaded before the app.
+ * You can add your own extra polyfills to this file.
+ *
+ * This file is divided into 2 sections:
```
Comment (KameshRajendran) [inline office-products/Export success/src/polyfills.ts:5]: what is the use of this file?

File: office-products/Export success/src/environments/environment.ts [hunk 8]
```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  production: false
+};
+
+/*
+ * For easier debugging in development mode, you can import the following file
```
Comment (KameshRajendran) [inline office-products/Export success/src/environments/environment.ts:10]: what is the use of this file?

File: office-products/Export success/src/app/app.component.ts [hunk 9]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import {
```
Comment (KameshRajendran) [inline office-products/Export success/src/app/app.component.ts:2]: **Folder name** 
**sample** - Common/ Load PDF, Excel, PPT file types
**web service** - Common/ web service 

------------
office-products/Export success/src/assets/.gitkeep - what is the use of this file?

**Review by KameshRajendran — CHANGES_REQUESTED** [review]

File: How to/Add water mark annotation in PDF viewer/Web Service/Controllers/PdfViewerController.cs (hunk 2 above)
Comment (SaravanaPriya31) [reply How to/Add water mark annotation in PDF viewer/Web Service/Controllers/PdfViewerController.cs:94]: added the method to get the ajax request after the load method , where to add this method is it need to add at the end ?

File: office-products/Export success/.browserslistrc (hunk 4 above)
Comment (SaravanaPriya31) [reply office-products/Export success/.browserslistrc:5]: This file is used by the build system to adjust CSS and JS output to support the specified browsers below. is it need to delete this file

**Pull request merged**